src/internal.rs

```diff
@@ -37,6 +41,79 @@ impl Default for FileTypes {
     }
 }
 
+#[derive(Clone, Copy, Debug, PartialEq)]
+enum SizeLimitType {
+    Max,
+    Min,
+}
+
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub struct SizeFilter {
+    size: u64,
+    limit_type: SizeLimitType,
+}
+
+// SI units for now
```

Comment:
Can this be removed? or maybe `// SI prefixes (powers of ten)` and `// Binary prefixes (powers of two)`?

Reply:
Yeah, forgot to change the comments when I updated